content/en/docs/concepts/storage/volumes.md

```diff
@@ -996,20 +996,23 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 {{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-The `CSIMigration` feature for `vsphereVolume`, when enabled, redirects all plugin operations
-from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this feature, the
+The `CSIMigrationvSphere` feature for `vsphereVolume` is enabled by default as of Kubernetes v1.25.
+All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver unless feature gates `CSIMigration` or `CSIMigrationvSphere` are disabled.
```

Comment:
I might be wrong, but I think that actually the `CSIMigration` feature gate will be graduated (locked to `true`) for v1.25. Check with SIG Storage folks to find out the actual situation with the related CSI driver migration feature graduations.

Reply:
checked https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#feature-2
`CSIMigration` is locked in 1.25.